api/src/test/java/org/openmrs/module/appointmentscheduling/api/AppointmentServiceTest.java

```diff
@@ -150,7 +150,7 @@ public void saveAppointment_shouldSaveEditedAppointment() throws Exception {
 	}
 
 	@Test
-	@Verifies(value = "should void given appointment", method = "voidAppointment(Appointment, String)")
+	@Verifies(value = "should delete given appointment", method = "voidAppointment(Appointment, String)")
```

Comment:
See my comments on TRUNK-2299, do we really want to change these, or just the display text?